elasticapm/transport/base.py

```diff
@@ -105,18 +105,17 @@ def __init__(
     def queue(self, event_type, data, flush=False):
         try:
             self._flushed.clear()
-            self._event_queue.put((event_type, data, flush), block=False, chill=not (event_type == "close" or flush))
+            if isinstance(self._event_queue, ChilledQueue):
```

Comment:
is this cheap enough to do each time / could this change during the runtime of the application?  Seems like this could be decided once at init and have a different queue action based onthat.

Reply:
good point, changed in e18a74a